ports/nrf/common-hal/busio/I2C.c

```diff
@@ -37,19 +37,21 @@
 #include "nrfx_spim.h"
 #include "nrf_gpio.h"
 
+// all TWI instances have the same max size
+// 16 bits for 840, 10 bits for 810, 8 bits for 832
+#define I2C_MAX_XFER_LEN         ( 1UL << TWIM0_EASYDMA_MAXCNT_SIZE)
```

Comment:
Should this be `( 1UL << TWIM0_EASYDMA_MAXCNT_SIZE) - 1`? That's my impression from reading the data sheet.

Reply:
yeah, that is right :D